lib/ModelMixins/AsyncMappableMixin.ts

```diff
@@ -49,3 +49,13 @@ export default function AsyncMappableMixin<
 
   return AsyncMappableMixin;
 }
+
+namespace AsyncMappableMixin {
+  export interface AsyncMappableMixin
+    extends InstanceType<ReturnType<typeof AsyncMappableMixin>> {}
+  export function isMixedInto(model: any): model is AsyncMappableMixin {
+    return model && model.isMappable;
```

Comment:
Ditto

Reply:
Recorded in https://github.com/TerriaJS/terriajs/issues/4522

Reply:
The change is from another PR and is being addressed in this issue - #4522